driver/driver_test.go

```diff
@@ -252,7 +252,7 @@ func TestStmt_ExecTooManyParams(t *testing.T) {
 		values[i] = int64(1)
 	}
 	_, err = stmt.Exec(values)
-	require.Errorf(t, err, "too many parameters (256) max = 255")
+	require.NoError(t, err)
```

Comment:
It would be nice to not only check that no error occurred, but that the parameters where actually correctly handled, i.e. 256 rows were inserted, with values ranging from 1 to 255.